kubeflow/core/centraldashboard.libsonnet

```diff
@@ -103,6 +88,9 @@ all(params):: [
       apiVersion: "v1",
       kind: "ServiceAccount",
       metadata: {
+        labels: {
+          app: "centraldashboard",
```

Comment:
Can you change this to name: "centraldashboard" as well since you changed the service